aat.tfvars

```diff
@@ -15,3 +15,5 @@ da_external_cert_name = "star-aat"
 external_cert_vault_uri = "https://infra-vault-nonprod.vault.azure.net/"
 
 asp_sku_size = "I3"
+
+managed_identity_object_id = "df18e9f9-5f50-46d8-b75f-34b856fcef24"
```

Comment:
Should this come from the vault?

Reply:
Nope, those are just resource IDs that are public

Reply:
no it's fine, they aren't credentials